src/locales/ko/move.ts

```diff
@@ -2937,7 +2937,7 @@ export const move: MoveTranslationEntries = {
   },
   bouncyBubble: {
     name: "생생버블",
-    effect: "물덩어리를 부딪쳐서 공격한다. 물을 흡수하여 데미지의 절반만큼 HP를 회복한다."
+    effect: "물덩어리를 부딪쳐서 공격한다. 물을 흡수하여 데미지의 절만큼 HP를 회복한다."
```

Comment:
wrong change here.
original text is right.

Reply:
Bouncy bubbles heals 100% of the damage dealt in Pokerogue and generations 8 and 9

Reply:
There is no word '절만큼' in korean.
If you want to explain 100% percent of damage, you can fix it like this

```suggestion
    effect: "물덩어리를 부딪쳐서 공격한다. 물을 흡수하여 데미지만큼 HP를 회복한다."
```